python/lsst/ts/ofc/ofc.py

```diff
@@ -84,7 +84,11 @@ def __init__(self, ofc_data: OFCData, log: logging.Logger | None = None) -> None
 
         self.ofc_data = ofc_data
 
-        self.state_estimator = StateEstimator(self.ofc_data)
+        self.state_estimator = StateEstimator(self.ofc_data, log=self.log)
+
+        self.zernike_controller = ZernikeController(
```

Comment:
Is this right? you need the ZernikeController here? Shouldn't this be in `set_controller`?